quiz/backend/helpers/tests/test_generate.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from helpers.gen_username.generate import generate_username
+
+
+def test_if_usernames_unique():
+    usernames = generate_username()
+
+    assert len(set(usernames)) == len(usernames)
+
+
+def test_generate_nondefault_num_usernames():
```

Comment:
You still have tests that are using files from the project, and it means you are not running tests in an isolated environment. Sorry for being pesky here, but the idea of unit tests is to test a for e. g. function without any distractions.